index.js

```diff
@@ -103,17 +103,18 @@ function run (cmd, cwd, env, cb) {
 
 function build (target, runtime, opts, cb) {
   var args = [
-    'rebuild',
-    '--target=' + target
+    'install',
+    '--target=' + target,
+    '--abi=' + abi.getAbi(target, runtime)
```

Comment:
Might need to add `--build-from-source`, if the `install` script is `node-gyp-build`.